src/components/StatusBadge/StatusBadge.stories.tsx

```diff
@@ -1,25 +1,28 @@
 import * as React from "react";
+
+import StatusBadge from "./StatusBadge";
 import bem from "../../utils/bem";
 import { withDesign } from "storybook-addon-designs";
 import { select, text } from "@storybook/addon-knobs";
```

Comment:
Instead of knobs, can you try out these stories using controls? https://storybook.js.org/docs/react/essentials/controls 

Storybook 6 recommends using controls so we're trying to not introduce any more code using knobs. If you'd like some examples of what those looks like within our codebase, Edwin's set up `Checkbox` to use controls.